Code/Framework/AzFramework/AzFramework/Spawnable/Script/SpawnableScriptAssetRef.cpp

```diff
@@ -26,10 +26,6 @@ namespace AzFramework::Scripts
                 ->Field("asset", &SpawnableScriptAssetRef::m_asset)
             ;
 
-            serializeContext->RegisterGenericType<AZStd::vector<SpawnableScriptAssetRef>>();
-            serializeContext->RegisterGenericType<AZStd::unordered_map<AZStd::string, SpawnableScriptAssetRef>>();
-            serializeContext->RegisterGenericType<AZStd::unordered_map<double, SpawnableScriptAssetRef>>(); // required to support Map<Number, SpawnableScriptAssetRef> in Script Canvas
```

Comment:
These lines support the SpawnableScriptAssetRef type into scriptcanvas containers, why do you need to remove them?

And add SpawnableScriptAssetRef into DataTrait is not really a scalable method.